packages/vscode-mdx/CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # Changelog
 
+## 1.2.0
+
+### Minor Changes
+
+- [#272](https://github.com/mdx-js/vscode-mdx/pull/272) [`4aad7ef`](https://github.com/mdx-js/vscode-mdx/commit/4aad7ef6ff16ab8e4695dcff344ebd3b1739f6bf) Thanks [@remcohaszing](https://github.com/remcohaszing)! - Support remark syntax plugins.
+
+  This extension supports remark syntax plugins.
+  Plugins can be defined in an array of strings or string / options tuples.
+  These plugins can be defined in `tsconfig.json` and will be resolved relative to
+  that file.
+
+  For example, to support [frontmatter][] with YAML and TOML and [GFM][]:
+
+  ```jsonc
+  {
+    "compilerOptions": {
+      // …
+    },
+    "mdx": {
+      "plugins": [["remark-frontmatter", ["toml", "yaml"]], "remark-gfm"]
+    }
+  }
+  ```
+
+  [frontmatter]: https://github.com/remarkjs/remark-frontmatter
+  [gfm]: https://github.com/remarkjs/remark-gfm
```

Comment:
I think this shows we can better not use link references in changesets.

Reply:
I don’t know what `this` refers to?

Reply:
The fact that the changesets are concatenated and added to a single changelog file. This means that if a reference is unique in a changeset, it might not be unique in the generated changelog.

Reply:
Ah, ok. Well, lint rules would catch that. But sure, feel free to use normal linkes